keep-ui/features/workflows/builder/lib/utils.tsx

```diff
@@ -26,7 +26,7 @@ const alertTriggerTemplate: V2StepTrigger = {
   id: "alert",
   properties: {
     alert: {
```

Comment:
i think we should change the internal representation of an alert trigger , something like 

```ts
properties: {
  alert: {
    cel: string;
    filters: Record<string, string>; // deprecated filters
  }
}
```

and the logic in `parser.ts` should be updated to parse deprecated filters into filters key https://github.com/keephq/keep/blob/e78953fe751fa678e80497074ecce5d201566315/keep-ui/entities/workflows/lib/parser.ts#L221-L229

Reply:
@Kiryous i'm not sure I follow on this comment, can you maybe elaborate?

> and the logic in parser.ts should be updated to parse deprecated filters into filters key

what does that mean?